packages/metadataform/src/form.tsx

```diff
@@ -38,27 +33,27 @@ export class FormWidget extends ReactWidget {
    */
   render(): JSX.Element {
     const formContext = {
+      defaultFormData: this._props.settings.default(),
       settings: this._props.settings,
       updateMetadata: this._props.metadataFormWidget.updateMetadata
```

Comment:
```suggestion
      defaultFormData: this._props.settings.default(),
      updateMetadata: this._props.metadataFormWidget.updateMetadata
```
Probably the *settings* can be removed from the `formContext` now there is the `defaultFormData`